internal/daysteps/daysteps.go

```diff
@@ -12,9 +19,52 @@ const (
 )
 
 func parsePackage(data string) (int, time.Duration, error) {
-	// TODO: реализовать функцию
+	// Создаём слайс dataInput из двух элементов типа string, создавая элементы из строки по разделителю ",".
+	// Также проверяем, что длина равна двум, чтобы в дальнейшем коде не выйти за границы слайса
+	dataInput := strings.Split(data, ",")
+	if len(dataInput) != 2 {
+		return 0, 0, fmt.Errorf("неверный формат данных")
+	}
+	// Первый элемент слайса - шаги. Преобразуем строку числа шагов в число, обрабатывая возможную ошибку, присваиваем значение переменной steps, ошибку - err
+	steps, err := strconv.Atoi(dataInput[0])
+	if err != nil {
+		return 0, 0, err
```

Comment:
Лучше оборачивать ошибки
```suggestion
		return 0, 0, fmt.Errorf("invalid steps format: %w", err)
```